nannou/src/window.rs

```diff
@@ -726,9 +726,10 @@ impl<'app> Builder<'app> {
         self
     }
 
+    #[cfg(not(target_os = "unknown"))]
```

Comment:
To understand this it's worth checking the implementation of `async_std::task_block_on`. The return type depends on whether your `target_os` is `"unknown"` or not.